src/vaadin-message.js

```diff
@@ -6,13 +6,14 @@
 import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
-
+import '@vaadin/vaadin-avatar';
```

Comment:
```suggestion
import '@vaadin/vaadin-avatar/src/vaadin-avatar.js';
```

Please use imports from `src` inside of the `src` folder.

Reply:
This seems to break styling for me, at least in the visual test package. Do I need to import styles somewhere else with this change?
![Screen Shot 2021-02-03 at 11 18 19](https://user-images.githubusercontent.com/525503/106725157-bad60f00-6611-11eb-816d-a34a34e36fbd.png)

Reply:
Yes, you need to update `theme/lumo/vaadin-messages.js and import the Lumo version of the avatar there:

```js
import '@vaadin/vaadin-avatar/theme/lumo/vaadin-avatar.js';
``` 

Same for the Material theme.

Reply:
Done.